packages/validator/package.json

```diff
@@ -27,7 +27,8 @@
     "format": "prettier -w .",
     "lint": "eslint . --ext .ts,.js",
     "lint:fix": "eslint --fix . --ext .ts,.js",
-    "check": "astro check && tsc --noEmit"
+    "check": "astro check && tsc --noEmit",
+    "clean": "rm -rf .turbo dist"
```

Comment:
This might not work on Windows if they don't have `rm`

I think use the nodejs package `rimraf` for this

Reply:
Noted!